src/main/flight/pid.c

```diff
@@ -288,7 +288,11 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         Kd[axis] = DTERM_SCALE * pidProfile->pid[axis].D;
     }
     dtermSetpointWeight = pidProfile->dtermSetpointWeight / 127.0f;
-    relaxFactor = 1.0f / (pidProfile->setpointRelaxRatio / 100.0f);
+    if (pidProfile->setpointRelaxRatio == 0) {
+        relaxFactor = 0; //avoid divide by zero
+    } else {
+        relaxFactor = 1.0f / (pidProfile->setpointRelaxRatio / 100.0f); // ie 1 to 100
```

Comment:
let's finally rewrite it to `100.f / pidProfile->setpointRelaxRatio`

Reply:
That's much nicer

Reply:
@DieHertz could you please implement that change, I've done it locally and pushed to my branch, I don't know how to rebase without stuffing things up...